ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -436,3 +478,18 @@ class BaremetalRedfishIPxeWholediskHttpLink(
     @utils.services('network')
     def test_ip_access_to_server(self):
         self.boot_and_verify_node()
+
+
+class BaremetalPXEIpmiDriverDirect(bsm.BaremetalStandaloneScenarioTest):
```

Comment:
Can check if this can be clubbed into single test case depending upon boot interface as ipxe or pxe.